cli_tools/gce_vm_image_import/importer/uefi_processor_test.go

```diff
@@ -0,0 +1,42 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUefiProcessor(t *testing.T) {
```

Comment:
```suggestion
func TestUefiProcessor_AddsUEFISuffixToPDName(t *testing.T) {
```

Reply:
Renamed to TestUefiProcessor_RecreateDisk since suffix is the result of recreating